Resources/Prototypes/_CP14/Entities/Objects/Tools/flashlight.yml

```diff
@@ -0,0 +1,40 @@
+- type: entity
+  name: Лазурный фонарь
```

Comment:
Prototypes required eng name and description + ru locales in /Locale/_CP14/_PROTO folder